crates/bevy_pbr/src/prepass/prepass_utils.wgsl

```diff
@@ -2,7 +2,7 @@
 
 #import bevy_pbr::mesh_view_bindings as view_bindings
 
-#ifndef DEPTH_PREPASS
+#ifdef DEPTH_PREPASS
```

Comment:
If this works I'm pretty sure that means we can just remove the ifdef entirely no?

The reason this was an ifndef is because the prepass_utils ended up being called _during_ the prepass for reasons I don't remember and this broke everything. 

I tried it locally and removing all them seems to still work as expected

Reply:
Leaving these in might make this safer to use? It would prevent people from using these functions when a given prepass isn't enabled.

Reply:
and will be required if making the bindings conditional (to free texture slots).